docs/Installation-Anaconda-Windows.md

```diff
@@ -101,14 +101,12 @@ activate ml-agents
 
 You should see `(ml-agents)` prepended on the last line.
```

Comment:
I believe the Anaconda Windows installation has been deprecated, so we probably don't need to update it.

Reply:
I think we should remove the doc then if it is deprecated.